runtime/bcutil/cfreader.c

```diff
@@ -2446,6 +2446,8 @@ j9bcutil_readClassFileBytes(J9PortLibrary *portLib,
 	UDATA syntheticFound = FALSE;
 	U_32 vmVersionShifted = flags & BCT_MajorClassFileVersionMask;
 	U_16 constantPoolAllocationSize = 0;
+	BOOLEAN requireIdentityObjectInterface = TRUE;
```

Comment:
we could remove this BOOLEAN and always add two extra slots to the constantpool. This would minimize the changes to the cfreader